src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (x) => {
```

Comment:
It is better not to give names to variables like `x`, `y`. I think it would be better to give names like `numberOne` and `numberTwo`.

Reply:
good hint!)